src/components/Header/Header.jsx

```diff
@@ -184,18 +194,34 @@ function Header({ isLogin }) {
                     </Tabs>
                 </Box>
             </Stack>
-            <Tooltip title="Open settings">
-                <Avatar
-                    alt="Your avatar"
-                    src={avartarImage}
-                    sx={{
-                        width: '2.8125rem',
-                        height: '2.8125rem',
-                        cursor: 'pointer',
-                    }}
-                    onClick={() => setOpenPf((prevState) => !prevState)}
-                />
-            </Tooltip>
+
+            {/* có đăng nhập hay chưa  */}
+            {isLogin ? (
+                <Tooltip title="Open settings">
+                    <Avatar
+                        alt="Your avatar"
+                        src={avartarImage}
+                        sx={{
+                            width: '2.8125rem',
+                            height: '2.8125rem',
+                            cursor: 'pointer',
+                        }}
+                        onClick={() => setOpenPf((prevState) => !prevState)}
+                    />
+                </Tooltip>
+            ) : (
+                <styleMui.containerButton>
+                    {loginRegister.map((vl) => (
+                        <styleMui.button
+                            key={vl.id}
+                            onClick={() => navigateFunc(vl.id)}
```

Comment:
I see you set an array with paths are `/login` and `/register` but you never use it. Let's use it instead of this function